Core/src/main/java/com/plotsquared/core/command/Done.java

```diff
@@ -94,7 +94,7 @@ public boolean onCommand(final PlotPlayer<?> player, String[] args) {
                 TagResolver.resolver("plot", Tag.inserting(Component.text(plot.getId().toString())))
         );
         final Settings.Auto_Clear doneRequirements = Settings.AUTO_CLEAR.get("done");
-        if (PlotSquared.platform().expireManager() == null || doneRequirements == null) {
+        if (doneRequirements == null || player.hasPermission(Permission.PERMISSION_ADMIN_COMMAND_DONE)) {
```

Comment:
Why is null check on the expire manager being removed??

Reply:
Because `PlotPlatform.expireManager` is marked as NotNull, so the check should not be required

Reply:
I've put the check back in as it seems sometimes not checked to be null and sometimes is